tests/Kernel/TenderRenderTest.php

```diff
@@ -0,0 +1,231 @@
+use Drupal\Tests\user\Traits\UserCreationTrait;
+use Drupal\user\Entity\User;
+use Symfony\Component\DomCrawler\Crawler;
+
+/**
+ * Tests the event rendering.
+ */
+class TenderRenderTest extends ContentRenderTestBase {
+
+  use UserCreationTrait;
+
+  /**
+   * {@inheritdoc}
+   */
+  protected static $modules = [
+    'field_group',
+    'composite_reference',
+    'oe_time_caching',
+    'oe_content_reference_code_field',
+    'oe_content_departments_field',
+    'oe_content_tender',
+    'oe_theme_content_tender',
+  ];
+
+  /**
+   * {@inheritdoc}
+   */
+  protected function setUp(): void {
+    parent::setUp();
+
+    $this->installConfig([
+      'composite_reference',
+      'oe_content_departments_field',
+      'oe_content_reference_code_field',
+      'oe_content_tender',
+      'oe_theme_content_tender',
+    ]);
+
+    module_load_include('install', 'oe_theme_content_tender');
+    oe_theme_content_tender_install();
+
+    module_load_include('install', 'oe_content');
+    oe_content_install();
+
+    // Set current user to UID 1, so that by default we can access everything.
+    $account = User::load(1);
+    $this->setCurrentUser($account);
+  }
+
+  /**
+   * Test an event being rendered as a teaser.
+   */
+  public function testTenderTeaser(): void {
+    // Create a Call for tender node.
+    /** @var \Drupal\node\Entity\Node $node */
+    $values = [
+      'type' => 'oe_tender',
+      'title' => 'Test Call for tenders node',
+      'oe_publication_date' => [
+        'value' => '2020-04-15',
+      ],
+      'oe_tender_opening_date' => [
+        'value' => '2020-04-30',
+      ],
+      'oe_tender_deadline' => [
+        'value' => date('Y') + 1 . '-06-10T23:30:00',
+      ],
+      'oe_reference_code' => 'Call for tenders reference',
+      'oe_departments' => ['http://publications.europa.eu/resource/authority/corporate-body/ABEC', 'http://publications.europa.eu/resource/authority/corporate-body/ACM'],
+      'uid' => 0,
+      'status' => 1,
+    ];
+    $node = Node::create($values);
+    $node->save();
+
+    $build = $this->nodeViewBuilder->view($node, 'teaser');
+    $html = $this->renderRoot($build);
+
+    // Check Open status label and background.
+    $assert = new ListItemAssert();
+    $expected_values = [
+      'title' => 'Test Call for tenders node',
+      'meta' => 'Call status: Open',
+      'image' => NULL,
+      'description' => new PatternAssertState(new FieldListAssert(), [
+        'items' => [
+          [
+            'label' => 'Reference',
+            'body' => 'Call for tenders reference',
+          ], [
+            'label' => 'Opening date',
+            'body' => '30 April 2020',
+          ], [
+            'label' => 'Deadline date',
+            'body' => sprintf('11 June %s, 09:30 (AEST)', date('Y') + 1),
+          ], [
+            'label' => 'Departments',
+            'body' => "Audit Board of the European Communities\nArab Common Market",
```

Comment:
Output has to be `Audit Board of the European Communities | Arab Common Market`

Reply:
Updated.